src/deploy/scope_managementgroup.ts

```diff
@@ -68,7 +67,11 @@ export async function DeployManagementGroupScope(azPath: string, region: string,
         core.info("Creating deployment...")
         var deploymentCode = await exec(`"${azPath}" deployment mg create ${azDeployParameters} -o json`, [], deployOptions);
         if (commandStdErr.trim().length !== 0) {
-            throw new Error(`Deployment process failed as some lines were written to stderr: ${commandStdErr}`)
+            if (failOnStdError.toLowerCase().trim() == "true") {
+                throw new Error(`Deployment process failed as some lines were written to stderr: ${commandStdErr}`)
+            } else {
+                core.error(commandStdErr)
+            }
         } else {
```

Comment:
Else should not be there anymore otherwise it will not check for exit code and will not parse outputs